### PR TITLE
work even on a readonly filesystem.

### PR DESCRIPTION
**Pull request by organic-ip**
I adapt scripts to have a special remount temporary in rw mode of directory to work on readonly root filesystem.

- [x] Add remount rw for hdd.xxx mount point and switch back to original mount status (ro / rw)
- [x] Add noatime for mount ram partition

### CODE REVIEW

File: log2ram [hunk 1]
```diff
@@ -10,19 +10,40 @@ isSafe () {
     [ -d $HDD_LOG/ ] || exit 1
 }
 
+remountRW() {
+    touch $HDD_LOG/$$ 2>/dev/null >/dev/null
+    RESU=$?
+    INITIAL_STATUS=ro
+    if  [ "$RESU" != "0" ] ; then
+	mount -o remount,rw ${HDD_LOG}
```
Comment (azlux) [inline log2ram:18]: can you fix indentation ?

File: log2ram [hunk 2]
```diff
@@ -74,17 +112,18 @@ case "$1" in
             RAM_LOG=$i
             HDD_LOG=$PATH_FIRST_PART/hdd.$PATH_LAST_PART
             LOG2RAM_LOG="${RAM_LOG}/${LOG_NAME}"
-        
-            [ -d $HDD_LOG/ ] || mkdir $HDD_LOG/
-            mount --bind $RAM_LOG/ $HDD_LOG/
+
+	    make_log_dir
```
Comment (azlux) [inline log2ram:116]: indentation

File: log2ram (hunk 1 above)
Comment (organic-ip) [reply log2ram:18]: Hello,
Yes I confirm your understanding.
My use case, is for my raspberry pi on sdcard, which work on readonly filesystem and I like to use ram2log to keep & synchronize punctually log to sdcard.

**Comment by azlux** [comment]
merged and fix indent into https://github.com/azlux/log2ram/commit/6d5b84ba75e240f072d3d57f16d9c15e4535812b

Thanks again for the pull.

Az

**Comment by azlux** [comment]
issue cause by this pull requests #113

**Pull request merged**